Sources/ReportEventType.swift

```diff
@@ -20,15 +22,17 @@ struct DeliveryEventType: ReportEventType {
     let event: String
     let publishId: String
     let deviceId: String
+    let userId: String
```

Comment:
Right now I'm setting `userId` to an empty string if the payload doesn't contain `userId`.
I can make this optional as well. What do you think @luismfonseca ?

Reply:
I think it makes more sense to have it as optional.